src/app/setup-wizard/pre-visit/review-pre-visit-setup/facility-details-summary/facility-details-summary.component.html

```diff
@@ -2,7 +2,7 @@ <h5 class="bold w-100 text-center">
     <fa-icon [icon]="faIndustry" class="me-2"></fa-icon> {{facility.generalInformation.name}}
 </h5>
 
-<table class="table table-bordered table-hover">
+<table class="table table-bordered table-hover table-sm">
     <tbody>
         <tr>
             <td class="w-25">
```

Comment:
same with company address, suggest to comment it out if not included in the alpha release